mobile-app/app/contexts/FeatureFlagContext.tsx

```diff
@@ -81,6 +81,7 @@ export function FeatureFlagProvider(
     return featureFlags.some((flag: FeatureFlag) => {
       if (
         flag.networks?.includes(network) &&
+        flag.app?.includes("MOBILE_LW") &&
```

Comment:
Just realized this would be a breaking change for previous versions of LW, do we need to handle it? Maybe something like `flag.app === undefined || flag.app?.includes("MOBILE_LW")` CC: @thedoublejay